### PR TITLE
Lab03ish

### PR DESCRIPTION
**Pull request by tektechnologies**

### CODE REVIEW

**Review by selmaT273 — COMMENTED** [review]
Looks good, however I am not seeing the modals rendering and am unsure why. I will be of more help as i catch up to lab 03 stuff and have a better eye in helping troubleshoot this.

**Comment by tektechnologies** [comment]
@selmaT273 you can find the link to the favorite foods page on the wookin page, on the left aside in a link to the favorite foods

**Comment by selmaT273** [comment]
Yep, it seems you've got code for the favorite foods to show up on a modal, however it does not appear to be displaying an individual image as a modal when it is clicked.

File: src/views/FavoriteFoodMain.js [hunk 1]
```diff
@@ -1,128 +1,48 @@
...
     this.state = {
-      gallery: jsonData
+      allFavoriteFoods: jsonData,
+      displayModal: false,
+      selectedFavoriteFood: {}
+     
     }
   }
   
+displayAsModel = (foodName) => {
```
Comment (selmaT273) [inline src/views/FavoriteFoodMain.js]: watch out for this typo in Modal

File: src/views/FavoriteFoodMain.js [hunk 2]
```diff
@@ -1,128 +1,48 @@
...
-      />
-      <FavoriteFoods
-      src = "https://picsum.photos/id/493/200/300"
-      title = "Milk and Cereal"
-      description = "Breakfast is the most important meal of the days."
+      
+       
+      <FoodMain 
+      allFavoriteFoods={this.state.allFavoriteFoods}
+      displayAsModal={this.displayAsModel}
```
Comment (selmaT273) [inline src/views/FavoriteFoodMain.js]: this.displayAsModel has the same typo in Modal.

File: src/views/FavoriteFoodMain.js (hunk 1 above)
Comment (tektechnologies) [reply src/views/FavoriteFoodMain.js]: Right on. I did not check that and I should have haah. @selmaT273

File: src/views/FavoriteFoodMain.js (hunk 2 above)
Comment (tektechnologies) [reply src/views/FavoriteFoodMain.js]: @selmaT273 thanks I updated let me know if you see anymore.

**Comment by selmaT273** [comment]
Looks good!

**Review by selmaT273 — APPROVED** [review]
Nice work!

**Pull request merged**